spaceship/lib/spaceship/portal/device.rb

```diff
@@ -59,9 +59,10 @@ def factory(attrs)
         end
 
         # @param mac [Bool] Fetches Mac devices if true
+        # @param includeDisabled [Bool] Whether to include disable devices. false by default.
```

Comment:
Everywhere you're using the `includeDisabled` parameter name, could you change it to `include_disabled` to be in-line with Ruby style? Thanks!